timed/settings.py

```diff
@@ -74,6 +74,9 @@ def default(default_dev=env.NOTSET, default_prod=env.NOTSET):
     "timed.subscription",
 ]
 
+if ENV == "dev":
```

Comment:
This is really set to `dev` in dev?

Reply:
Yes, it's written in `README.md`

>## Development

To get the application working locally for development, make sure to create a file `.env` with the following content:

```
ENV=dev
DJANGO_OIDC_CREATE_USER=True
````

Reply:
:+1: